### PR TITLE
Rs 2022 05 15

### PR DESCRIPTION
**Pull request by rschlaikjer**
Retro review

### CODE REVIEW

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 1]
```diff
@@ -613,12 +623,13 @@ You're not a picky eater, right, Tohno-kun?"
...
 -- Page 104, Offset 743.
 -- 先輩の笑顔には悪意が一片もない。
-There is not a trace of malice in her smile.
+There isn't a trace of malice in her smile.
 }
 [sha:98c8132fadeb41c8862d82f095636a9be65ea1a5]{
 -- Page 104, Offset 744.
 -- 何かの冗談とか嫌がらせではなく、一点の曇りもないパーフェクトな善意なのだった。
-She is not joking or teasing, she has perfectly good intentions without a hint of evil.
+She isn't joking around or pulling prank on me here,
```
Comment (yk1512) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:631]: pulling **a** prank

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 2]
```diff
@@ -1019,22 +1038,22 @@ I look at Arihiko silently.
...
 -- Page 140, Offset 827.
 -- 「引っ越したら誰だって転校生になるわけじゃありません。住所が変わっただけで、学校は変わってないんです。
-"Moving doesn't make anyone a new student. I've just changed my address, not my school.
+"Moving doesn't necessarily mean I'm going to transfer. I'm just changing addresses, not schools.
 }
 [sha:70bdc8094db70f9f55cfad28815c92703df90b6e]{
 -- Page 140, Offset 828.
 -- 今までは親戚の家にいたんですけど、今日からもとの家に戻るだけなんですよ」
-I've been living with my relatives, but from today onwards, I'm going back to my old home."
+I've been living with my relatives up til now, but from today on I'll be back at my old house."
```
Comment (yk1512) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:1056]: from today, I'll be back

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 3]
```diff
@@ -1185,7 +1204,8 @@ I said as I eat the curry rice, without looking at Arihiko's face.
 [sha:440d41bd8b7967f5fb6360fe99a97b39d7cc08f4]{
 -- Page 156, Offset 858.
 -- アレで文句を言ったらバチが当たるってもんか」
-If you complained about that, I would've hit you."
+// Reading as Shiki would suffer divine punishment if he were to complain about his time with the Arimas?
+Lighting would strike you down if you were to complain about the time you spent there."
```
Comment (yk1512) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:1208]: I think it is divine punishment yeah

'You'd be struck by lighting if you complained about the time you spent there'

File: script/Ciel/Day 1/01_01_CIEL01_3_3_1.txt [hunk 4]
```diff
@@ -143,27 +142,27 @@ Despite her words, Senpai has an embarrassed, but happy grin on her face.
...
 -- Page 227, Offset 1025.
 -- これは怒っている……んじゃないよな、きっと。
-Surely... she's not angry, right?
+I'm... %{i}pretty%{/i} sure she's not actually angry.
 }
 [sha:0dddc5bae7c3604dec69b9085dba0e9259951bd8]{
 -- Page 228, Offset 1026.
 -- 「わたし、がぜん遠野くんのコトが知りたくなっちゃいました。
-"I suddenly want to know more about you.
+"I've taken somewhat of a sudden interested in you, Tohno-kun.
```
Comment (yk1512) [inline script/Ciel/Day 1/01_01_CIEL01_3_3_1.txt:150]: interest

File: script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt [hunk 5]
```diff
@@ -82,17 +81,20 @@ I just want to hear it."
 [sha:1a53784082080f7b88457ca712e88fca16222eb0]{
 -- Page 239, Offset 1056.
 -- 自分の家に戻ると言っていましたけど、今までご実家には戻られていなかったんですか？」
-You said you were returning to your own home, but you've never gone back to your family home?"
+You said you were returning to your old house, does that mean you've never been back to your family home?"
```
Comment (yk1512) [inline script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt:84]: I feel like a 'but' makes the sentence read more smoothly

**Review by yk1512 — DISMISSED** [review]

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 6]
```diff
@@ -1,17 +1,19 @@
 [sha:6443efefa2105c6359d66c3013de730225ecc467]{
 -- Page 53, Offset 621.
 -- 昼休み、廊下は行き交う生徒たちで雑然としている。
-The corridor is crowded with students passing by during lunch break.
+It's lunch break, and the corridor is filled with a jumble of students heading this way and that.
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:4]: headed this way and that

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 7]
```diff
@@ -1,17 +1,19 @@
...
 }
 [sha:6719eda1ee308917e8db69d000aa7886362514ff]{
 -- Page 53, Offset 622.
 -- 学食に向かう生徒が大半だが、中には弁当を片手に友人のいるクラスにお邪魔したり、中庭や体育館に向かう生徒もいる。
-Most of them head for the cafeteria, but some visit their friends' classes with their lunch boxes in hand or head for the courtyard or the gym.
+Most of them are on their way to the cafeteria,
+ but some are going to visit their friends' classes with their lunch boxes in hand,
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:10]: with lunchboxes in hand

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 8]
```diff
@@ -132,27 +133,28 @@ It was fun talking with Ciel-senpai.
...
 -- そのお詫びです。つかぬ事をお訊きしますけど、遠野くんはこれからお昼ごはんですか？」
-I apologize for that. If you don't mind me asking, are you going to have lunch now?"
+That's what I'm apologizing for.
+ If you don't mind me asking, were you on your way to get lunch?"
 }
 [sha:a6c3e278a1f3633155b54789b7c355854f6e3be2]{
 -- Page 65, Offset 650.
 -- 「ですよ。食堂に行くか購買で済ませるか迷っていたところです」
-"Yes. I was trying to decide if I should go to the cafeteria or just buy something."
+"Yes. I was trying to decide whether I should go to the cafeteria or just buy something."
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:152]: mild preference for previous version

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 9]
```diff
@@ -132,27 +133,28 @@ It was fun talking with Ciel-senpai.
...
 }
 [sha:985a8841f7849e0511df0d27adc49973a54dd6f7]{
 -- Page 66, Offset 651.
 -- 「よかった、それならご一緒できますね。朝のお詫びにおごっちゃいますから、これから食堂に行きましょう」
-"Great, then I can join you. I'll treat you as an apology for this morning, so let's go to the cafeteria now."
+"Great, then let me join you. Let's go to the cafeteria, I'll treat you to make up for what happened this morning."
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:157]: cafeteria. I'll

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 10]
```diff
@@ -408,7 +415,9 @@ That sounds all too familiar.
 [sha:9b9709551e9a085143ede236fc9c453bb6e2be88]{
 -- Page 90, Offset 702.
 -- 「眼鏡の似合うショートカットで、元気よく動き回って、それでいて頼れる空気だしまくり？」
-"A short-haired girl who looks good with glasses. Cheerful and unable to keep still, but dependable nonetheless?"
+"A short-haired girl who looks good with glasses,
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:418]: sentence is quite run-on and duplicate "still" makes it a little stilted. Here's an alt. suggestion but I'm not that happy with the word "restless".
> A short-haired girl who looks good in glasses and is both restless and cheerful, but %{i}still%{/i} comes off as dependable.

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 11]
```diff
@@ -458,12 +467,12 @@ In her hands, she carries a tray for two people.
...
 -- Page 93, Offset 712.
 -- 先輩はごく自然に席に座った。
-Senpai takes a seat quite naturally.
+Senpai nonchalantly takes a seat at the table.
 }
 [sha:85cb718dde4931c485109ec77c0a869bd8a70cfd]{
 -- Page 93, Offset 713.
 -- 俺の対面―――即ち、有彦のとなりの席である。
-She takes the seat opposite me―――which puts her next to Arihiko.
+She chooses the seat opposite me―――which puts her next to Arihiko.
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:475]: opposite me―――putting her next to Arihiko.

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 12]
```diff
@@ -473,7 +482,8 @@ She takes the seat opposite me―――which puts her next to Arihiko.
 [sha:0c9f579b68093bc8eafe6fc97860c65043f80fb4]{
 -- Page 94, Offset 715.
 -- よく分からない声をあげる有彦。おそらく突然の幸運にうまく対応できないのだろう。気持ちは分かる。何故ならさっきまでの俺がそうだった。
-Arihiko makes an unfamiliar sound. Perhaps he can't cope with the sudden good fortune. I know how he feels because that's what I was experiencing not too long ago.
+Arihiko makes an unfamiliar sound. Perhaps he can't cope with the sudden good fortune.
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:485]: his sudden good fortune

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 13]
```diff
@@ -969,12 +988,12 @@ I see. In other words, she's just an airhead, isn't she?
...
 -- Page 136, Offset 815.
 -- 「俺の家もこの街ですよ。徒歩にして30分弱。
-"My house is in this city too. It takes less than thirty minutes to walk there.
+"My house is in the city too. It's about a thirty minute walk from here.
 }
 [sha:14446076934f405b58e13da7f0c09ce17186a54f]{
 -- Page 136, Offset 816.
 -- ほら、北口から郊外にあがると、坂の上に住宅地があるでしょ。あそこの奥まで行ったところです」
-If you go up to the suburbs from the north gate, you'll see a residential area at the top of a hill. It's at the end of that."
+If you head up into to the suburbs from the north gate, there's that residential area on top of the hill, right? It's at the end of that."
```
Comment (Avuxo) [inline script/Ciel/Day 1/01_01_CIEL01_2_3.txt:996]: north gate, you know that residential area on top of the hill? It's

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 1/01_01_CIEL01_2_3.txt:1056]: I prefer with "on"

File: script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 3 above)
Comment (Avuxo) [reply script/Ciel/Day 1/01_01_CIEL01_2_3.txt:1208]: I prefer the stronger image from "strike you down"

**Review by Avuxo — APPROVED** [review]
otsukare

**Pull request merged**